interface/patient_file/encounter/Tests/Service/ViewHelperTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
see discussion above regarding licensing. since we do not need to save trees (ie. this is not paper), would recommend sticking with the standard licensing/copyright header (if wish to change it, then we should discuss that on the forum to get community input).